### PR TITLE
feat: handle remaining translations

### PR DESCRIPTION
**Pull request by flagrede**
## Description

Handle remaining translations.

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [ ] provided a link to the relevant issue or specification
- [ ] provided instructions on how to test
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### How to test

1.

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] verified React components follow DRY principles
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

File: web-components/src/components/form/form.mock.ts [hunk 1]
```diff
@@ -12,4 +14,6 @@ export const bottomTextMapping = {
   countryLabel: 'Country',
   stateLabel: 'State / Region',
   postalCodeLabel: 'Postal Code',
+  locationStatePlaceholderLabel: 'State / Region',
```
Comment (blushi) [inline web-components/src/components/form/form.mock.ts:17]: what's the difference between this and `stateLabel`?

**Review by blushi — APPROVED** [review]
LGTM pre-approving just a couple comments

File: web-components/src/components/cards/OrderSummaryCard/OrderSummaryCard.tsx [hunk 2]
```diff
@@ -4,19 +4,37 @@ import { OrderSummaryImage } from './OrderSummaryCard.Image';
 import { OrderSummaryProps } from './OrderSummaryCard.types';
```
[the remark this answers is not in the document]
Comment (flagrede) [reply web-components/src/components/cards/OrderSummaryCard/OrderSummaryCard.tsx:4]: Ok, well, reverting this file would also revert some others. I think applying these changes to the moved file might be easier. Let me know if you need any help with that.

File: web-components/src/components/form/form.mock.ts (hunk 1 above)
Comment (flagrede) [reply web-components/src/components/form/form.mock.ts:17]: The mock is incorrect; it should be `Please choose a state,` I'll update it, thanks.

**Pull request merged**